<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GCSFD Version 3 - Now featuring GUI &amp; SQLite!</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QPushButton" name="button_delete">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="button_add">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="display_tab">
      <property name="toolTip">
       <string>Persons</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="person_tab">
       <attribute name="title">
        <string>Persons</string>
       </attribute>
       <widget class="QComboBox" name="search_by_dropdown">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gender</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Date of Birth</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Date of Death</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="search_field">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>222</x>
          <y>20</y>
          <width>211</width>
          <height>21</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>search input here...</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="DisplayTable">
        <property name="geometry">
         <rect>
          <x>-5</x>
          <y>50</y>
          <width>461</width>
          <height>311</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Gender</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Date Of Death</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="label_search_for_person">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Search for:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_sort_by_person">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>0</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Sort by:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_sort_by_person">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gender</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Date of Birth</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Date of Death</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="computer_tab">
       <attribute name="title">
        <string>Computers</string>
       </attribute>
       <widget class="QTableWidget" name="DisplayTable_2">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>50</y>
          <width>461</width>
          <height>311</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Year built</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>built?</string>
         </property>
        </column>
       </widget>
       <widget class="QComboBox" name="search_by_dropdown_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="search_field_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>232</x>
          <y>20</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>search input here...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_search_for_computer">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Search for:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_sort_by_computer">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>0</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Sort by:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_sort_by_computer">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Build Year</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Type</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Built?</string>
         </property>
        </item>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>466</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
